### PR TITLE
[4.0] No jQuery in the xml files...

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
- Added a new method in core.js
```js
  /**
   * Method that resets given inputs and submits the given form
   * @param {HTMLElement}  The element that initiates the call
   */
  Joomla.resetFilters = (element) => { }
```
- Changed all the xml filters to use this method
- The method will reset all the fields **of the filters form only** except:
-- the task
-- the token
-- the boxcheked

### Testing Instructions
Go to Admin templates, check if the site/admin dropdown resets the table options
screenshot 2018-12-31 at 21 53 28

### Documentation Changes Required
Yes.
Basically to implement the filter reset one just needs to add in the XML:
```xml
		onchange="Joomla.resetFilters(this)"
```
That simple

### CODE REVIEW

**Comment by infograf768** [comment]
Change missing for filter_modulesadmin.xml

**Comment by wilsonge** [comment]
Looks good to me :)
